.gitignore

```diff
@@ -25,4 +25,5 @@ coverage/
 
 npm-debug.log*
 yarn-debug.log*
-yarn-error.log*
\ No newline at end of file
+yarn-error.log*
+yarn.lock
```

Comment:
I think adding `yarn.lock` in the `.gitignore` is a good idea to avoid changing it on main. Thank you 👍